src/main/rx/crsf.h

```diff
@@ -45,7 +45,10 @@ enum {
     CRSF_FRAME_GPS_PAYLOAD_SIZE = 15,
     CRSF_FRAME_VARIO_SENSOR_PAYLOAD_SIZE = 2,
     CRSF_FRAME_BATTERY_SENSOR_PAYLOAD_SIZE = 8,
-    CRSF_FRAME_BAROMETER_ALTITUDE_PAYLOAD_SIZE = 2,
+    CRSF_FRAME_BAROMETER_ALTITUDE_PAYLOAD_SIZE = 2,       // 0x09 Altitude only
+    CRSF_FRAME_BAROMETER_ALTITUDE_VARIO_PAYLOAD_SIZE = 3, // 0x09 Altitude + Vario (TBS CRSF standard)
+    // CRSF_FRAME_BAROMETER_ALTITUDE_VARIO_PAYLOAD_SIZE = 4,   // 0x09 Altitude + Vario (EdgeTX / ELRS historical standard)
```

Comment:
As this can break things. There should be a parameter to allow the legacy way of sending this packet for 2 major versions of INAV. This will allow any incompatible systems a grace period to get up to spec. 

I suggest a `crsf_use_legacy_baro_packet` which defaults to `off`. **Off** uses the new correct to spec packet. Whereas **on** uses the previous implementation. A note should be included in the documentation (Settings.md is fine) stating that this is deprecated and will be removed in INAV 11.